modules/microshift-firewall-apply-settings.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-firewall-applying-settings_{context}"]
+== Applying firewall settings
+After you have finished configuring, run the following command to restart the firewall and apply settings:
+
+[source,terminal]
+----
+$ sudo firewall-cmd --reload
+[source,terminal]
+----
+
+[id="microshift-firewall-verifying-settings_{context}"]
+== Verifying settings
```

Comment:
Rather than having this as a separate module/procedure, we could just include it in the previous procedure. Up to you if you would want to do that.

Reply:
I think I got pinged on this on an earlier review and was required to break it out. I see your point, but I'll leave it for now.